src/main/java/de/upb/crypto/math/structures/groups/exp/SmallExponentPrecomputation.java

```diff
@@ -66,4 +79,30 @@ public void compute(int windowSize) {
             }
         }
     }
+
+    public void computeNegativePowers(int windowSize) {
```

Comment:
Can we have two versions of this method and let the `Group` choose which one to use? (1) this one and (2) one that inverts the positive powers?

I feel like we're giving up some performance here by completely recomputing the values from scratch using n group ops instead of n inversions (in groups where inversions are faster than group operations).

Reply:
Good idea, I have computed inversion to group op ratios for each of our `GroupImpl` implementations and will be (or am) using this to let the `Group` make that decision.